### ISSUE TITLE
FeaturesExtractor:could not parse file common/utils/encode-decode.js with error 'missed some imaginary tokens:

### ISSUE THREAD

**Issue by smola** [src-d/style-analyzer#180]
I get this warning:

```
WARNING:4af0:FeaturesExtractor:could not parse file common/utils/encode-decode.js with error 'missed some imaginary tokens: "
    /<form[^>]*>/,
    "', skipping
```

The problem seems to be here: https://github.com/jlouiss/freeCodeCamp/blob/ff07a3637863ed15439acab642818d6e368d3c08/common/utils/encode-decode.js#L23

**Comment by zurk** [comment]
ok, I found the reason. https://github.com/bblfsh/javascript-driver/issues/37, for now, I add a workaround. Sorry, @m09 for stealing your issue :)

**Comment by m09** [comment]
This is normally handled by
```
            if (node.token or node.start_position and node.end_position
                    and node.start_position != node.end_position and not node.children):
                node_tokens.append(node)
```
This case fails for some other reason if I didn't miss anything.

**Comment by zurk** [comment]
the problem is in a comment here. Because the comment is a child of regexp node:
for this code
```js
x = // comment
  /<regexp>/;
```

UAST gives:
```
#           Token       Internal Role         Roles Tree                                                    
                                                                                                            
(1, 1, 12)  ||          File                  FILE                                                          
(1, 1, 12)  ||          Program               ┣ MODULE                                                      
(1, 1, 12)  ||          ExpressionStatement   ┃ ┣ STATEMENT                                                 
(1, 1, 11)  ||          AssignmentExpression  ┃ ┃ ┣ EXPRESSION, ASSIGNMENT, OPERATOR, BINARY                
(1, 1, 2)   |x|         Identifier            ┃ ┃ ┃ ┣ EXPRESSION, IDENTIFIER, ASSIGNMENT, BINARY, LEFT      
(2, 1, 11)  ||          RegExpLiteral         ┃ ┃ ┃ ┣ EXPRESSION, LITERAL, REGEXP, ASSIGNMENT, BINARY, RIGHT
(1, 5, 15)  | comment|  CommentLine           ┃ ┗ ┗ ┗ ┗ COMMENT                                             
(1, 5, 15)  | comment|  CommentLine           ┗ COMMENT  
```
